plugins/net-forward.yaml

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
No other plugin (with the exception of krew itself) currently includes a license in the manifest. It also doesn't include the copyright owner. I'd recommend to just remove it, so that it will be subject to the license of `krew-index`.

Reply:
Removed.